### PR TITLE
Better tablet styles

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
## QA

Have a look on tablet device or use browser responsive tools to select an appropriate size for a tablet. Click around and have a look at things to ensure it all works as expected and the styles are usable.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7762/

File: jujugui/static/gui/src/app/components/env-switcher/list/_list.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
   box-sizing: border-box;
   width: $width;
   margin-left: -20px;
+  z-index: 1;
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/env-switcher/list/_list.scss]: This should probably use the z-index function

**Comment by hatched** [comment]
When visiting jaas not logged in to /new there is a `/` beside the logo.

screen shot 2017-06-06 at 8 50 14 am

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7770/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7771/

File: jujugui/static/gui/src/app/assets/css/_tablet.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    &__content {
+      display: block;
+      top: 116px;
+      overflow-y: scroll;
+    }
+
+    &__column{
+      display: block;
+      width: 100%;
+      display: block;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/assets/css/_tablet.scss]: double display

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7776/

**Comment by hatched** [comment]
Thanks @Lukewh :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**